staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -175,9 +176,6 @@ const maxWait = 10 * time.Second
 func (q *delayingType) waitingLoop() {
 	defer utilruntime.HandleCrash()
 
-	// Make a placeholder channel to use when there are no items in our list
-	never := make(<-chan time.Time)
-
```

Comment:
suggest something like this instead (a local timer is easier to reason about than a member variable):

```go
// Make a placeholder channel to use when there are no items in our list	
never := make(<-chan time.Time)

// Make a timer that expires when the item at the head of the waiting queue is ready
var nextReadyAtTimer *time.Timer
```